ygot/render_test.go

```diff
@@ -2344,6 +2596,12 @@ func (*uFieldMulti) IsU() {}
 
 func TestUnionInterfaceValue(t *testing.T) {
 
+	// This is the only test that is used by the simple union API -- it is
```

Comment:
Does this comment mean that this test case is only used by the simple union API, or the simple union API only tests this case?

Reply:
This is meant to say that the function under test, `unionPtrValue`, is mostly used for the wrapper union API -- the only simple union API that uses this function is the unsupported type.

I'll update the test function name and this comment to make this clearer.